### PR TITLE
DL-12767 company-registration-frontend - dependency upgrades

### PR DESCRIPTION
**Pull request by arunankannan**
!!! :guardsman: PLEASE UPDATE THE TEXT BELOW AND REMOVE THIS LINE :guardsman: !!!

# Descriptive Title - including Jira story or Github issue number

**Bug fix / New feature** (delete as appropriate)

Please include a summary / description of the change and which issue it fixes.  Include any relevant user needs and context.

## Checklist

* [x] I've included appropriate tests with any code I've added
* [x] I've executed the acceptance test pack locally to ensure there are no regressions
* [x] I've added my code using logical, atomic commits, squashing as appropriate - including the Jira issue number in the commit message
* [x] I've run a dependency check to ensure all dependencies are up to date

### CODE REVIEW

File: build.sbt
Comment (derickdamoah) [inline build.sbt:1]: scala version can be increased to 2.13.13. Also can you create an it / test folder and move all the integration test folders in there as highlighted here https://github.com/hmrc/sbt-settings#integration-tests

File: conf/application.conf [hunk 1]
```diff
@@ -281,13 +253,6 @@ microservice {
 
 auditing {
   enabled=true
-  traceRequests=true
-  consumer {
```
Comment (derickdamoah) [inline conf/application.conf:285]: why were the auditing config values removed? they are used here -> https://github.com/hmrc/company-registration-frontend/blob/237137b328d73da00801b5dd4e338ce6a0db3494/it/itutil/IntegrationSpecBase.scala#L52

File: project/AppDependencies.scala [hunk 2]
```diff
@@ -5,35 +5,29 @@ import sbt._
...
-    "commons-validator"       %   "commons-validator"               %  "1.7",
-    "uk.gov.hmrc"             %%  "play-language"                   % s"6.1.0$playVersion",
-    "uk.gov.hmrc"             %%  "play-frontend-hmrc-play-28"              % s"8.5.0",
+    "uk.gov.hmrc"             %% s"play-partials$playVersion"                 % "9.1.0",
+    "uk.gov.hmrc"             %% s"http-caching-client$playVersion"           % "11.2.0",
+    "uk.gov.hmrc"             %% s"play-conditional-form-mapping$playVersion" % "2.0.0",
+    "org.bitbucket.b_c"       %   "jose4j"                                    %  "0.9.6",
+    "commons-validator"       %   "commons-validator"                         %  "1.8.0",
+    "uk.gov.hmrc"             %%  "play-language"                             %  "7.0.0",
+    "uk.gov.hmrc"             %%  s"play-frontend-hmrc$playVersion"           % bootsrapVersion
```
Comment (derickdamoah) [inline project/AppDependencies.scala:24]: this can be increased to 9.10.0 https://github.com/hmrc/play-frontend-hmrc

File: project/AppDependencies.scala
Comment (derickdamoah) [inline project/AppDependencies.scala:1]: play version can be increase to 30
bootstrap version can be increased to 8.6.0 -> https://github.com/hmrc/bootstrap-play (there's also a spelling error)
hmrc-mongo can be increased to 1.9.0 -> https://github.com/hmrc/hmrc-mongo
play-frontend-hmrc can be increased to 9.10.0 https://github.com/hmrc/play-frontend-hmrc

Comment (arunankannan) [reply project/AppDependencies.scala:1]: this is special case where we can not increase the play version to 3.0
the way this app is implemented, there are url parameters that are longer than the allowed size limit 2040, enforced by play3.0. so it was decided that we differ this upgrade of play3.0 for this app.
all other libs version are in-accordance with play 2.9

File: build.sbt
Comment (arunankannan) [reply build.sbt:1]: sure, done this now. thanks

File: conf/application.conf (hunk 1 above)
Comment (arunankannan) [reply conf/application.conf:285]: ok sure, have put this back. thanks

**Review by derickdamoah — APPROVED** [review]

**Pull request merged**